### DISCUSSION TITLE
gagal dalam ploting dengan geom_line

### DISCUSSION THREAD

**Discussion by tatapinandhika** [teamalgoritma/community#229] [Q&A] [answered]
Saya ingin menampilkan plot pertumbuhan penjualan suatu produk dengan menggunakan geom_line, tetapi output yang didapatkan tidak sesuai. Mungkin ada yang bisa bantu, dimana letak kesalahan saya. Terimakasih
<img width="1440" alt="Screen Shot 2022-07-28 at 10 04 31" src="https://user-images.githubusercontent.com/101564223/181404633-26f7ce28-bc5b-45c5-b5de-5c3c02e76252.png">
<img width="1440" alt="Screen Shot 2022-07-28 at 10 04 37" src="https://user-images.githubusercontent.com/101564223/181404654-f0d5ead6-3c61-4442-8faa-84615624e371.png">

**Comment by shinddayo** [comment]
@tatapinandhika coba nggak pakai petik di x dan y. jadi aes(x=year, y=total)

**Reply by tatapinandhika** [reply]
makasi sebelumnya, tapi kenapa kosong ya sekarang
<img width="1440" alt="Screen Shot 2022-07-28 at 19 39 27" src="https://user-images.githubusercontent.com/101564223/181496592-e09aec15-abf0-4d3b-89c7-76091dbfd68f.png">

**Comment by idoali** [comment]
Selamat malam pak. Mungkin bisa diperbesar garisnya, contohnya seperti ini `geom_line(size=10)`. Semoga dapat membantu pak.

**Reply by tatapinandhika** [reply]
masih tidak kelihatan mas.
<img width="1440" alt="Screen Shot 2022-07-29 at 20 07 19" src="https://user-images.githubusercontent.com/101564223/181755399-64ca1c6a-6f3e-4a0c-8c87-e1e27c3165b8.png">

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @tatapinandhika , melanjutkan dari pengerjaan Bapak di atas, geom line tidak terbentuk karena data x nya bertipe data character. geom line mengira bahwa tiap tahun adalah satu entitas (grup) terpisah sehingga line nya tidak disambungkan. 

Untuk mengatasinya, silahkan set parameter `group = 1` di geom_line agar dia membuat satu garis saja mengikuti nilai sumbu x.

```
ggplot(data, aes(x=year, y=total) +
    geom_line(group = 1)
```

Semoga dapat membantu!
